crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -78,43 +74,54 @@ def repositories(platform)
       def admin_ip
         NodeObject.admin_node.ip
       end
+
+      def web_port
+        Proposal.where(barclamp: "provisioner").first.raw_attributes["web_port"]
+      end
     end
 
     def initialize(platform, repo)
       @platform = platform
       @id = repo
-      @registry = Repository.registry[@platform]["repos"][@id]
+      @config = Repository.registry[@platform]["repos"][@id]
       @url = url
     end
 
+    def is_remote?
```

Comment:
Style/PredicateName: Rename `is_remote?` to `remote?`. (https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)